File: src/main/java/xyz/terrific/utils/jframe/TableModel.java
package xyz.terrific.utils.jframe;

import javax.swing.table.AbstractTableModel;

/**
 * @author TerrificTable55
 */
public class TableModel extends AbstractTableModel {

    @Override
    public int getRowCount() {
        return 0;
    }

    @Override
    public int getColumnCount() {
        return 0;
    }

    public boolean isCellEditable(int row, int column){
        return false;
    }

    @Override
    public Object getValueAt(int rowIndex, int columnIndex) {
        return null;
    }

}

File: src/main/java/xyz/terrific/utils/jframe/TableModel.kt
package xyz.terrific.utils.jframe

import javax.swing.table.AbstractTableModel

/**
 * @author TerrificTable55
 */
class TableModel : AbstractTableModel() {
    override fun getRowCount(): Int {
        return 0
    }

    override fun getColumnCount(): Int {
        return 0
    }

    override fun isCellEditable(row: Int, column: Int): Boolean {
        return false
    }

    override fun getValueAt(rowIndex: Int, columnIndex: Int): Any? {
        return null
    }
}
